src/utils/components/StatusBox/StatusBox.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Alert, AlertVariant } from '@patternfly/react-core';
+
+import Loading from '../Loading/Loading';
+
+type StatusBoxProps = {
+  error?: any;
+  loaded: boolean;
+};
+
+const StatusBox: FC<StatusBoxProps> = ({ children, error, loaded }) => {
```

Comment:
i dont like the name statusbox, its not meaningful, maybe LoadingOrError ? or select something more creative :)